core/domain/platform_parameter_registry.py

```diff
@@ -443,3 +443,35 @@ def _to_platform_parameter_model(
     platform_parameter_domain.DataTypes.NUMBER,
     default=0.2
 )
+
+Registry.create_platform_parameter(
+    ParamName.SYSTEM_EMAIL_ADDRESS,
+    'Email address used for system issued actions',
+    platform_parameter_domain.DataTypes.STRING,
+    # TODO(ankita240796): Update post the first trial release of these params.
```

Comment:
Do not use username in TODO, create an issue and reference it here.

Reply:
Done.